charts/virtual-kubelet/values.yaml

```diff
@@ -3,8 +3,6 @@ image:
   tag: latest
   pullPolicy: Always
 
-## `provider` should be one of aws, azure, azurebatch, etc...
```

Comment:
Sorry, I might have missed something but why are we taking the provider out of the chart?

Reply:
This doesn't remove the provider. This provides what I personally think is a better error message if you don't specify a provider. The current error message:

```
error calling eq: invalid type for comparison
```

This PR changes the error message to this:

```
You must specify a Virtual Kubelet provider
```